cognite/client/_api/datapoints.py

```diff
@@ -165,7 +172,7 @@ def retrieve_latest(
         return DatapointsList._load(res, cognite_client=self._cognite_client)
 
     def query(
-        self, query: Union[DatapointsQuery, List[DatapointsQuery]]
+        self, query: Union[DatapointsQuery, List[DatapointsQuery]], ignore_unknown_ids=False
```

Comment:
Should be on the `DatapointsQuery` instead.

Reply:
I think it is prefered by users as is, but we can ask them